packages/itwinui-css/src/workflow-diagram/classes.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './index';
+
+.iui-workflow-diagram {
+  @include iui-workflow-diagram;
+}
+
+.iui-workflow-diagram-step {
+  @include iui-workflow-diagram-step;
+}
+
+.iui-workflow-diagram-track-content {
+  @include iui-stepper-track-content;
+}
+
+.iui-workflow-diagram-circle {
+  @include iui-workflow-diagram-circle;
```

Comment:
This `-circle` name doesn't make sense. It's making me wonder if we even need this as a separate element. Is it possible to apply these styles on the parent `<li>`?

Reply:
I was able to remove an extra class but I also had to stop reusing a mixin from `stepper.scss`.  Renamed `-circle` to `-content`.
```html
<li class="iui-workflow-diagram-step">
  <span class="iui-workflow-diagram-content">Start</span>
</li>
```